src/components/card/Description.vue

```diff
@@ -262,6 +262,8 @@ export default {
 
 #description-preview {
 	min-height: 100px;
+	width: max-content;
+	overflow-x: scroll;
```

Comment:
Haven't tried out yet, but would `auto` also work maybe to not always have a scorllbar?